x-pack/plugins/ml/public/util/theme_colors.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import chrome from '../../../../../src/legacy/ui/public/chrome';
+const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+
+export const euiColorForTheme = (euiColor) => {
```

Comment:
This function seemed nice and generic. I had no idea where it should live so I put in ML for now so I could reuse it. Any ideas @cchaos?

Reply:
We can ask the platform team if they know a good spot to put this a service anyone can grab. But I'd name it more specifically too like `getEuiThemeColorsForJS`

Reply:
Oh I see, you pass back the the actual color value, not just the object.... Maybe even make it more generic then like `euiColor`